charts/aws-ebs-csi-driver/templates/_node-windows.tpl

```diff
@@ -1,5 +1,6 @@
 {{- define "node-windows" }}
-{{- if .Values.node.enableWindows }}
+{{- if (.Values.node).enableWindows }}
+---
```

Comment:
This `---` is just for styling yes?

Reply:
No, it's for separating multiple YAML documents in the same file. It used to be inside `node.yaml` and `node-windows.yaml` but I moved it here so it's only rendered if the relevant `DaemonSet` is enabled, otherwise extra `---`s will be rendered when the `DaemonSet` is disabled.